### DISCUSSION TITLE
Annotator version for API

### DISCUSSION THREAD

**Discussion by tkmamidi** [KarchinLab/open-cravat#243] [Usability & Feature Requests]
Hi,
Is there a way to specify annotator version from API to query data? This will prevent our downstream pipeline from failing cause of version changes.
Thanks!

**Comment by kmoad** [comment]
Only one version of an annotator can be installed at a time. The run.opencravat.org website is kept up-to-date with the latest version of all annotators.

If you need a specific version, you'd have to use a local installation. Then the command to specify the version to install is 
```
oc module install ModuleName --version 1.2.3
```

To see what versions of a module are available, and the history of changes, run

```
oc module info ModuleName
```

**Comment by tkmamidi** [comment]
Is there a way you can provide version information for data sources in the JSON output?
https://open-cravat.readthedocs.io/en/latest/API.html#single-variant-annotation
